config/plugins/visualizations/jqplot/jqplot_bar/src/script.js

```diff
@@ -3,7 +3,7 @@ import _ from "underscore";
 import jqplot from 'jqplot-exported/jqplot';
 import { LineRenderer } from 'jqplot-exported/LineRenderer';
 import { BarRenderer } from 'jqplot-exported/plugins/BarRenderer';
-import { OHLCRenderer } from 'jqplot-exported/plugins/OHLCRenderer';
+import { OHLCRenderer } from 'jqplot-exported/plugins/OhlcRenderer';
```

Comment:
prettier isn't happy with this, do we need to exclude the file ?

Reply:
@mvdbeek Prettier shouldn't change or complain about this, I'll take a look.

Reply:
It wasn't prettier; it was the ci test cache failing; rebase fixed it.